.circleci/config.yml

```diff
@@ -46,7 +46,7 @@ jobs:
 
       - run:
           name: debugging missing tests
-          command: ls -R junit
+          command: ls -R shields/junit
```

Comment:
👍

Reply:
I just had a major facepalm moment 🤦‍♂️, really hope this is all it was the whole time